### PR TITLE
Fixed quest log not closing on escape press.

### PR DESCRIPTION
**Pull request by Spencer0**
### Issue #1033
Currently, when a user opens the quest log via shift clicking the QuestieTracker, they are unable to close it using the escape button after. This pull request address that.

### CODE REVIEW

File: Modules/QuestieTracker.lua [hunk 1]
```diff
@@ -200,7 +200,7 @@ local function _ShowQuestLog(Quest)
             QuestLogExFrameMaximizeButton:GetScript("OnClick")(QuestLogExFrameMaximizeButton)
         end
     else
-        QuestLogFrame:Show()
+       ToggleQuestLog();
```
Comment (BreakBB) [inline Modules/QuestieTracker.lua]:
```suggestion
        ToggleQuestLog();
```

Comment (BreakBB) [reply Modules/QuestieTracker.lua]: Such an easy fix, thanks for this!

Comment (Spencer0) [reply Modules/QuestieTracker.lua]: Anytime! Glad to contribute to something so useful, i'll be around more now that i finally hit 60.

Comment (BreakBB) [reply Modules/QuestieTracker.lua]: Haha, sounds good! We appreciate any help we get. @AeroScripts and I are around level 30 😅

**Pull request merged**